inc/views/base/xmlhttp/buddyselect.twig

```diff
@@ -0,0 +1,51 @@
+<table border="0" cellspacing="1" cellpadding="4" class="tborder" style="width: 300px;">
+    <thead>
+        <tr>
+            <td class="thead">
+                <div class="float_right" style="margin-top: 3px;">
+                    <span class="smalltext">
+                        <a href="javascript:void(0)" onclick="UserCP.closeBuddySelect(true); return false;">{{ lang.close }}</a>
+                    </span>
+                </div>
+                <div><strong>{{ lang.select_buddies }}</strong></div>
+            </td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td class="trow1 smalltext">{{ lang.select_buddies_desc }}</td>
+        </tr>
+        <tr>
+            <td class="trow2">
+                <div style="height: 200px; overflow: auto;" >
+                    <table border="0" cellspacing="1" cellpadding="4" class="tborder" style="border: 0; width: 270px;" align="right">
+                        {% for buddy in buddies %}
+                            {% if buddy.lastactive > ('now'|date('U') - mybb.settings.wolcutoff) and (buddy.invisible == 0 or mybb.user.usergroup == 4) and buddy.lastvisit != buddy.lastactive %}
```

Comment:
Indenting here seems to have got a little confused.

Reply:
For real! Ahahahah my bad, guess many PRs are cursed by indenting issues.